### PR TITLE
update imaging_parameters.py to handle line names with underscores

### PR DESCRIPTION
**Pull request by apmtowner**
Changed lines 2996-3010 to allow for line names with underscores in them in the line_imaging_parameters_custom dictionary. There are also a bunch of filename changes in that dictionary - those are specific to my current work and can be ignored as they will be reset before final imaging anyway.

### CODE REVIEW

File: reduction/imaging_parameters.py [hunk 1]
```diff
@@ -2987,23 +2987,26 @@
...
     "W51-IRS2_B6_12M_robust0_sio": {
-        "threshold": "9.6mJy",  # typical rms is 2.7-3.2 mJy, using 3sigma for threshold (14 Dec. 2020)
-        "startmodel": "W51-IRS2_B6_uid___A001_X1296_X187_continuum_merged_12M_robust0_selfcal8_finaliter",
+        "threshold": "6.4mJy",  # typical rms is 2.7-3.2 mJy, using 3sigma for threshold (14 Dec. 2020)
+        "startmodel": "W51-IRS2_B6_uid___A001_X1296_X187_continuum_merged_12M_robust0_selfcal8",
     },
 }
 
 
+linelist = ["h41a","ch3cnv8=1","13cs_2-1","n2hp","ch3cch_62-52","h2cs_312-211","oc33s_18-17","sio","h2co_303-202","c18o","so_6-5","12co","ocs_19-18","13cs_5-4","h30a"]
```
Comment (keflavich) [inline reduction/imaging_parameters.py:2996]: just to check: is `linellist` the same as the keys in `default_lines`?  If so, we should just move that up here, and do "for linename in default_lines"

Comment (apmtowner) [reply reduction/imaging_parameters.py:2996]: Yes, it is the same.

**Comment by keflavich** [comment]
oops I shouldn't have merged this, it's still broken

**Comment by keflavich** [comment]
ok now should be ok

**Pull request merged**